src/javascript/app/pages/user/set_currency.js

```diff
@@ -51,10 +51,15 @@ const SetCurrency = (() => {
                 $('#fiat_currencies').setVisibility(1);
                 $('#fiat_currency_list').html(fiat_currencies);
             }
-            const crytpo_currencies = $cryptocurrencies.html();
-            if (crytpo_currencies) {
+            const crypto_currencies = $cryptocurrencies.html();
+            if (crypto_currencies) {
                 $('#crypto_currencies').setVisibility(1);
-                $('#crypto_currency_list').html(crytpo_currencies);
+                $('#crypto_currency_list').html(crypto_currencies);
+            }
+            if ((!fiat_currencies && crypto_currencies) || (fiat_currencies && !crypto_currencies)) {
```

Comment:
I think we could also add this to a constant? perhaps `const has_currency_selected = ...`